pubspec.yaml

```diff
@@ -11,7 +11,7 @@ environment:
   sdk: '>=2.3.0 <3.0.0'
 
 dependencies:
-  analyzer: '>=0.38.0 <0.39.0'
+  analyzer: '>=0.38.3 <0.39.0'
```

Comment:
How about `^0.38.3`?

Reply:
Good idea. Though I've got three other PRs that make this same change, so I'll wait to merge those first to keep things easier on me.